src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
That isn't an answer to YAGNI. Your breaking consistency with existing code and need a reason for this.